### ISSUE TITLE
hi，I‘d ask about the framework support depthwise separable convolution?

### ISSUE THREAD

**Issue by parquets** [Raukk/tf-keras-surgeon#8]
as the title, In Keras-surgeon, I see an issue about it, do the framework  support depthwise separable convolution? Will you updata the coda in long?

**Comment by goodboyanush** [comment]
Hi, this fork of `tf-keras-surgeon` has support for 
 - DepthwiseConv2D
 - LSTM, RNN, GRU
 - updated BatchNormalization

https://github.com/s36srini/tf-keras-surgeon

**Comment by Raukk** [comment]
I'm sorry, between work and life, I have not had the time to maintain this project.
